code/modules/mob/living/carbon/human/human.dm

```diff
@@ -936,6 +937,17 @@
 			var/msg = span_notice("[key_name_admin(usr)] has removed [key_name(src)] from purrbation.")
 			message_admins(msg)
 			admin_ticket_log(src, msg)
+	//SKYRAT EDIT ADDITION BEGIN - CRYO SEND
+	if(href_list[VV_HK_SEND_CRYO])
+		if(!check_rights(R_SPAWN))
+			return
+		src.send_to_cryo()
```

Comment:
DM already implicitly does the `src.` here, you don't need to explicitly add it.
```suggestion
		send_to_cryo()
```